src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -119,7 +119,6 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter('sulu_media.ghost_script.path', $config['ghost_script']['path']);
 
         // storage
-        $container->setParameter('sulu_media.media.max_file_size', '16MB');
```

Comment:
strange parameter injected into the MediaManager but never used 🙈 set it to the same value as `sulu_media.upload.max_filesize` beneath

Reply:
Any reason to keep it at all then? Do you want to keep it for BC reasons? If yes I would at least add some comment in here, so that we remember removing it at some point.